cryptography

```diff
@@ -0,0 +1,15 @@
+---
+Title: 'Cryptography' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'A method of protecting information and communications through the use of codes that allow only the sender and receiver of a message to view its contents.' # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - 'Cryptography provides confidentiality, integrity, authentication for sensitive information while it is stored (at rest), traveling across a network (in transit), and existing in memory (in use).Two main types of cryptosystems enforce confidentiality: symmetric and asymmetric cryptosystems.'
+  - 'A second subject name'
+  - 'An nth subject name'
```

Comment:
```suggestion
Subjects:
  - 'A second subject name'
  - 'An nth subject name'
```

The first item under `Subjects` seems more sensible as the opening text of the entry. For the remaining items, let's replace with one or more appropriate subjects from [this list](https://github.com/Codecademy/docs/blob/main/documentation/subjects.md).